Plandomizer.py

```diff
@@ -527,6 +537,31 @@ def fill(self, window, worlds, location_pools, item_pools):
         locations = {}
         if self.locations:
             locations = {loc: self.locations[loc] for loc in random.sample(self.locations.keys(), len(self.locations))}
+        for starting_item, record in pattern_dict_items(self.starting_items):
```

Comment:
This isn't working for me; `pattern_dict_items` expects the values to be records with `item` but the starter record only has `count`. And I think it wants the keys to be locations or location patterns... Should we just remove the call?

Reply:
Not sure how that ended up there. Definitely a poor copy/paste job since it is the very similar as the normal location fill, but I had something that worked there previously. Updated the code to something that seems to work as expected.